README

```diff
@@ -224,23 +224,43 @@ Usage: cewl [OPTIONS] ... <url>
 	-c, --count: Show the count for each word found.
 	-v, --verbose: Verbose.
 	--debug: Extra debug information.
-      
+
 	Authentication
 	--auth_type: Digest or basic.
 	--auth_user: Authentication username.
 	--auth_pass: Authentication password.
-      
+
 	Proxy Support
 	--proxy_host: Proxy host.
 	--proxy_port: Proxy port, default 8080.
 	--proxy_username: Username for proxy, if required.
 	--proxy_password: Password for proxy, if required.
-      
+
 	Headers
 	--header, -H: In format name:value - can pass multiple.
-      
+
     <url>: The site to spider.
 
+Using Docker
+========
+
+Create container for CeWL
+```sh
+sudo docker build -t "greenmind/cewl:1" .
+```
+
+Using container
+```sh
+sudo docker run -it --rm "greenmind/cewl:1" -h
+```
+
+Linking to container
+```sh
+sudo docker run -it --rm -v "${PWD}:/root" "greenmind/cewl:1" -h
+```
+> Exemple use: sudo docker run -it --rm -v "${PWD}:/root" "greenmind/cewl:1" http://businesscorp.com.br -w /root/saida.txt
```

Comment:
```suggestion
> Example use: sudo docker run -it --rm -v "${PWD}:/root" "greenmind/cewl:1" http://businesscorp.com.br -w /root/saida.txt
```

Reply:
Dude it's a typo, patch it or leave it but don't ask for someone to patch it... xD
Sudo is not needed _if the current user is in the docker group_, but I wouldn't advise that as this implies that the current user has a trivial privilege escalation to be root on the box. So yeah, better keep the sudo here :)